hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -99,6 +101,7 @@ def log_parameters(safe_dir, resolution, radiometry, scale, speckle_filter, dem_
     log.info(f'    Include RGB decomposition : {include_rgb}')
     log.info(f'    Orbit file                : {orbit_file or "Original Predicted"}')
     log.info(f'    Output name               : {product_name}')
+    log.info(f'    DEM name                  : {dem_name}')
```

Comment:
`DEM strategy` or something similar might be a better label here?  This is before we've prepared the DEM, so we don't yet know the specific DEM that legacy processing will use.

Reply:
Yeah, this was kinda the struggle I was having yesterday  -- I like DEM strategy, but I'm not sure what we'd call the opposite of `legacy`. 

We've got a specify DEM, or "the best of a set", which I can't figure out how to explain well concisely

Reply:
Can punt this for a future discussion though

Reply:
I'm comfortable leaving this as-is, literally echoing the value of the `dem_name` input, which is what all the other lines in this function are doing